How-to-Install-SCF.md

```diff
@@ -207,8 +216,11 @@ To install SCF
   To invoke the tests run the command
    ```
    kubectl create \
-      --namespace="${NAMESPACE}" \
+      --namespace=scf \
       --filename="kube/cf/bosh-task/smoke-tests.yml"
```

Comment:
Is there a way to do this other than changing the yml file? I don't see anything obvious in the docs. I guess this is why we want helm for this instead.